cmd/init.go

```diff
@@ -0,0 +1,109 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	v1 "github.com/cosmos/atlas/server/router/v1"
+	"github.com/urfave/cli/v2"
+)
+
+var manifestTemplate *template.Template
+
+func init() {
+	var err error
+	tmpl := template.New("manifestFileTemplate").Funcs(template.FuncMap{
+		"StringsJoin": strings.Join,
+	})
+	if manifestTemplate, err = tmpl.Parse(defaultManifestTemplate); err != nil {
+		panic(err)
+	}
+}
+
+func InitCommand() *cli.Command {
+	return &cli.Command{
+		Name:  "init",
+		Usage: `Initialize a manifest.`,
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:    "dir",
+				Aliases: []string{"d"},
+				Usage:   "The directory to generate the manifest in",
+			},
+		},
+		Action: func(ctx *cli.Context) error {
+			var (
+				buffer   bytes.Buffer
+				manifest v1.Manifest
```

Comment:
Yikes, this is probably a codesmell if we need to import the router package here. I wonder if we should move the `Manifest` type elsewhere or just write the raw template to file?

Why do we even need to use a template and call `Execute`?

Reply:
Wrote the string to the file.